svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/(file-view)/[...tree]/+page.svelte

```diff
@@ -3,25 +3,27 @@
   import FileExplorerContainer from '$lib/components/FileExplorer/FileExplorerContainer/FileExplorerContainer.svelte';
   import FileExplorerNav from '$lib/components/FileExplorer/FileExplorerNav/FileExplorerNav.svelte';
   import FileExplorerReadme from '$lib/components/FileExplorer/FileExplorerReadme/FileExplorerReadme.svelte';
-  import type { LayoutServerData } from './$types';
-
-  export let data: LayoutServerData;
-  const { repoInfo, username, repo } = data;
-  const { description, website, tags, readme, tree, activeBranch } = repoInfo;
+  import type { PageServerData } from './$types';
+  export let data: PageServerData;
 </script>
 
 <div class="container grid grid-cols-12 subpage">
   <section class="col-span-9 col-sm-span-12">
     <FileExplorerNav />
-    <FileExplorerContainer contents={tree} branch={activeBranch} {username} {repo} />
+    <FileExplorerContainer
+      folder={data?.folder}
```

Comment:
None of these properties should be `undefined` or `null` (except `data.userInfo` which isn't used in this template) so there should be no need for the optional chaining operator `?.` for any of these